src/containers/user-profile/profile-info/ProfileContainerMobile.tsx

```diff
@@ -8,6 +10,17 @@ import AppChipList from '~/components/app-chips-list/AppChipList'
 import ProfileDoneItemsList from '~/components/icon-with-text-list/ProfileDoneItemsList'
 
 import { styles } from '~/containers/user-profile/profile-info/ProfileInfo.styles'
+import { UserResponse } from '~/types'
+
+interface ProfileContainerMobileProps {
+  actionIcon: ReactNode
+  accInfo: ReactNode
+  buttonGroup: ReactNode
+  defaultQuantity: number
+  doneItems: number
```

Comment:
doneItems is not a number, it's an array of objects { title: string, description: string }